cmd/proxy/actions/app.go

```diff
@@ -75,12 +75,18 @@ func App(conf *config.Config) (*buffalo.App, error) {
 		app = app.Group(prefix)
 	}
 
-	// Register exporter to export traces
-	exporter, err := observ.RegisterTraceExporter(conf.TraceExporterURL, Service, ENV)
+	// RegisterExporter is used for registering and exporting traces from opencensus
+	// The error from the RegisterExporter would be nil if the tracer was specified by the user and
+	// and the trace exporter was created successfully.
+	// RegisterExporter returns a function that needs to be run for all services(datadog, jaeger)
```

Comment:
Can you update the comment as such
RegisterExporter returns the function that all traces are flushed to the exporter and the exporter needs to be stopped. The function should be called when the exporter is no longer needed.